<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="openGL">
         <property name="text">
          <string>Use hardware-accelerated drawing (OpenGL)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPaths">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Map/Tileset search paths:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="pathsList"/>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="pathUpButton">
               <property name="toolTip">
                <string>Move directory up</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="editor.qrc">
                 <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="pathDownButton">
               <property name="toolTip">
                <string>Move directory down</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="editor.qrc">
                 <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="addPathButton">
               <property name="toolTip">
                <string>Add directory</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset resource="editor.qrc">
                 <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="removePathButton">
               <property name="toolTip">
                <string>Remove directory</string>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
               <property name="icon">
                <iconset resource="editor.qrc">
                 <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>pathUpButton</zorder>
            <zorder>addPathButton</zorder>
            <zorder>removePathButton</zorder>
            <zorder>pathDownButton</zorder>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pathsList</tabstop>
  <tabstop>pathUpButton</tabstop>
  <tabstop>pathDownButton</tabstop>
  <tabstop>addPathButton</tabstop>
  <tabstop>removePathButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
